Detectors/gconfig/src/GConfLinkDef.h

```diff
@@ -16,4 +16,8 @@
 #pragma link off all functions;
 #pragma link C++ class o2::SimSetup+;
 
+#pragma link C++ class  o2::conf::ConfigurableParam+;
```

Comment:
this line is probably not needed as its done somewhere else?

Reply:
No, that has been moved from `Detectors/Passive/src/PassiveLinkDef.h` as I thought this might be a more general place to put it? We can also move it somewhere else?

Reply:
I would prefer not to make such implicit changes that have nothing to do with the PR.